### PR TITLE
update codecs parameter to latest spec

### PR DESCRIPTION
**Pull request by cconcolato**
Related to #9 
- add still_picture field
- clarify that first operating point is used
- add support for extensibility if we want to add other operating points.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -375,21 +375,23 @@ All fields following the sample entry 4CC are expressed as double digit decimals
...
 
-The colourPrimaries, transferCharacteristics, matrixCoefficients and videoFullRangeFlag parameter values SHALL equal the value of matching fields in the [=Sequence Header=], if color_description_present_flag is set to 1, otherwise the fields can take any value.
+The colourPrimaries, transferCharacteristics, matrixCoefficients and videoFullRangeFlag parameter values SHALL equal the value of matching fields in the [=Sequence Header=], if color_description_present_flag is set to 1, otherwise they SHOULD not be set, defaulting to the values below. The videoFullRangeFlag is represented by a single digit.
 
-For example, codecs="av01.2.01.10.0.112.09.16.09.01" represents AV1 profile 2, level 1, 10-bit YUV content, non-monochrome, with 4:2:0 chroma subsampling, ITU-R BT.2020 primaries, ST 2084 EOTF, ITU-R BT.2020 non-constant luminance color matrix, and full-range chroma/luma encoding.
+For example, codecs="av01.2.0.01.10.0.112.09.16.09.1" represents AV1 profile 2, level 1, 10-bit YUV content, non-monochrome, with 4:2:0 chroma subsampling, ITU-R BT.2020 primaries, ST 2084 EOTF, ITU-R BT.2020 non-constant luminance color matrix, and full-range chroma/luma encoding.
```
Comment (tomfinegan) [inline index.bs]: The example text needs an update:

AV1 profile 2, non-still, level 1,

File: index.bs [hunk 2]
```diff
@@ -412,4 +414,6 @@ All the other fields are optional, mutually inclusive (all or none) fields. If n
 </tr>
 </table>
 
-The string codecs="av01.01.01.08" in this case would represent AV1 profile 1, level 1, 8-bit YUV content with 4:2:0 chroma subsampling, ITU-R BT.709 color primaries, transfer function and matrix coefficients, and luma/chroma encoded in the "limited" range.
+The string codecs="av01.1.0.01.08" in this case would represent AV1 profile 1, level 1, 8-bit YUV content with 4:2:0 chroma subsampling, ITU-R BT.709 color primaries, transfer function and matrix coefficients, and luma/chroma encoded in the "limited" range.
```
Comment (tomfinegan) [inline index.bs]: Need still here, no?

- codecs="av01.1.0.01.08."
- AV1 profile 1, non-still, level 1,

**Comment by tdaede** [comment]
Looks OK, with tomfinegan's changes.

File: index.bs (hunk 1 above)
Comment (cconcolato) [reply index.bs]: updated.

File: index.bs (hunk 2 above)
Comment (cconcolato) [reply index.bs]: updated.

**Pull request merged**